### PR TITLE
Update PIA - make build and wrapper update save

### PR DESCRIPTION
**Pull request by julianu**
Make the build.sh and pia.sh save for future updates and resolve the issues in #44546

### CODE REVIEW

**Comment by julianu** [comment]
@BiocondaBot please add label

File: recipes/pia/meta.yaml [hunk 1]
```diff
@@ -13,7 +13,7 @@ build:
     - {{ pin_subpackage('pia', max_pin="x") }}
 
 source:
-  url: https://github.com/medbioinf/pia/releases/download/1.4.8/pia-1.4.8.zip
+  url: https://github.com/medbioinf/pia/releases/download/1.4.9/pia-1.4.9.zip
```
Comment (mencian) [inline recipes/pia/meta.yaml:16]:
```suggestion
  url: https://github.com/medbioinf/pia/releases/download/{{ version }}/pia-{{ version }}.zip
```

**Review by mencian — COMMENTED** [review]
A suggestion to use the `version` variable in the source URL, otherwise looks good to me!

File: recipes/pia/meta.yaml (hunk 1 above)
Comment (rpetit3) [reply recipes/pia/meta.yaml:16]: agreed

**Review by bgruening — APPROVED** [review]

**Pull request merged**